src/ResourceManager/Resources/Commands.Resources/Extensions/ResourceExtensions.cs

```diff
@@ -15,65 +15,13 @@
 using System.Collections.Generic;
 using System;
 
-#if !NETSTANDARD
-using Microsoft.Azure.Management.Resources;
-using Microsoft.Azure.Management.Resources.Models;
-#else
 using Microsoft.Azure.Management.ResourceManager;
 using Microsoft.Azure.Management.ResourceManager.Models;
-#endif
 
 namespace Microsoft.Azure.Commands.Resources.Models
 {
     public static class ResourceExtensions
     {
-#if !NETSTANDARD
-        public static DeploymentExtended Deployment(this DeploymentGetResult deployment)
```

Comment:
these aren't used anywhere in the desktop code?

Reply:
Not since I moved us away from Microsoft.Azure.Management.Resources to Microsoft.Azure.Management.ResourceManager.